src/main/java/core/basesyntax/dao/FruitTransactionDao.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.dao;
+
+import java.util.List;
+
+public interface FruitTransactionDao {
+    List<String[]> getAllTransactions(String fileName);
```

Comment:
DAO conception is used to access DB. We do not store Transactions in the DB, so, this interface is not correct